features/environment.py

```diff
@@ -37,6 +38,12 @@
 from features.steps.sh_run import run
 from features.steps.util import create_new_venv
 
+NO_PYTHON38 = "no_python38"
+
+
+def _is_python38(scenario):
+    return NO_PYTHON38 in scenario.tags and "3.8" in platform.python_version()
```

Comment:
Maybe as more robust?
```suggestion
    return NO_PYTHON38 in scenario.tags and sys.version_info[:2] == (3, 8)
```